src/components/user/Avatar/index.js

```diff
@@ -1,106 +1,36 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-import { bodies, DefaultBody } from "./parts/bodies";
```

Comment:
Those are not used anymore and can be removed

Reply:
👍